editor/src/clj/editor/app_view.clj

```diff
@@ -2430,24 +2430,24 @@ If you do not specifically require different script states, consider changing th
 
 (handler/defhandler :copy-project-path :global
   (active? [app-view selection evaluation-context]
-           (context-resource-file app-view selection evaluation-context))
+           (context-resource app-view selection evaluation-context))
```

Comment:
The difference between `context-resource` and `context-resource-file` is that `context-resource` also returns folder resources, while `context-resource-file` only returns file resources.